pkg/db/migration/000019.go

```diff
@@ -0,0 +1,132 @@
+package migration
```

Comment:
Hi @jvallesm,
I placed the old Go migration script in the convert/xxx package because sometimes we might have structs with the same name in the migration package. Keeping them in different packages helps us avoid this problem.

For example, in different migration steps, we might have structs named `recipe` but with different content.

Reply:
Yeah I assumed that was the reason so I wanted to check on this. I see the point there, my experience so far with migrations was that I didn't (or barely) need to define structs for the conversion so a dedicated package wasn't required and having the migration code next to the SQL scripts added clarity. In my mind, in case of conflict, we could have structs such as `recipe000019`.

But I have no strong opinion about using the `migration` package vs `convert000019`, and we have code already there so I'll move the Go migration there.

Do you agree that using `migration.go` to centralise the step->migration code makes sense (instead of `cmd/migration/main.go`?

Reply:
> Do you agree that using migration.go to centralise the step->migration code makes sense (instead of cmd/migration/main.go?
Yeah, I think that's a good change.